### PR TITLE
Init a CONTRIBUTING.md [ci skip]

### PR DESCRIPTION
**Pull request by JuanitoFatas**
This Pull Request adds a contributing guidelines document to help others set up local dev environment and running tests.

[Preview CONTRIBUTING.md](https://github.com/JuanitoFatas/retrodot/blob/8270e6a/CONTRIBUTING.md).

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -18,10 +18,18 @@ chdir APP_ROOT do
...
-  # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
-  # end
+  puts "\n== Copying sample files =="
+  unless File.exist?('config/database.yml')
+    cp 'config/database.yml.sample', 'config/database.yml'
+  end
+
+  unless File.exist?('.env')
+    cp '.env.sample', '.env'
```
Comment (reidmix) [inline bin/setup]: it's clearly undocumented, but we are using `configurerator` and are now using `.env.development` and we should have changed `.env.sample` to `.env.development.sample` -- everything else looks 👍

**Comment by reidmix** [comment]
hey thanks for the contributing the contributing.md doc, that's pretty darn cool and helps us!

File: bin/setup (hunk 1 above)
Comment (JuanitoFatas) [inline bin/setup]: Hi I updated to copy from `.env.development.sample` to `.env.development` and removed unused `.env.sample`. 😊

**Comment by lexelby** [comment]
Thanks for contributing!! :D Survey question time: how did you find retrodot?

**Comment by JuanitoFatas** [comment]
> Survey question time: how did you find retrodot?

Was checking out heroku's GitHub yesterday, retrodot was the most-recently updated project. :)

**Comment by lexelby** [comment]
Neato! :) We're in our first iteration, looking to produce an MVP that gathers together some disparate pieces of automation around incident retrospectives.  We (Heroku SRE) are looking to add a lot more functionality to Retrodot in the future, so stay tuned!

**Comment by JuanitoFatas** [comment]
> an MVP that gathers together some disparate pieces of automation around incident retrospectives. We (Heroku SRE) are looking to add a lot more functionality to Retrodot in the future, 

That's awesome!

> so stay tuned!

I stay watched!

screenshot 2016-08-25 22 09 00

**Comment by jmervine** [comment]
👍 on merging this @lexelby and/or @joshuatobin, but I'll leave it to you.

**Comment by JuanitoFatas** [comment]
> @JuanitoFatas mind rebasing to prep?

rebased 😊

**Pull request merged**